apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -333,6 +335,40 @@ public EmulatorDetection EmulatorDetection
             }
         }
 
+        /// <summary>
+        /// Other known clients may set an additional value for the x-goog-api-client version header. This
+        /// array contains the only valid values.
+        /// </summary>
+        private static readonly string[] s_allowedVersionHeaders = { "efcore" };
+
+        private string ValidateVersionHeader(string header)
+        {
+            GaxPreconditions.CheckNotNullOrEmpty(header, nameof(header));
```

Comment:
We should allow it to be set to null IMO, given that's the default value. (Setting a property to its current value shouldn't be an error.)

Reply:
Sounds reasonable to me, I'll update that.